manifests/ml-training-on-eks/core-modules.yaml

```diff
@@ -127,7 +127,7 @@ parameters:
           #   - mitigate-log4shell
 ---
 name: fsx-lustre
-path: git::https://github.com/awslabs/idf-modules.git//modules/storage/fsx-lustre?ref=main&depth=1 
+path: git::https://github.com/awslabs/idf-modules.git//modules/storage/fsx-lustre?ref=release/1.4.0&depth=1 
```

Comment:
@a13zen The latest changes for FSx-Lustre as we have discussed have NOT been released on IDF yet....this version is not correct.   I can release IDF now...but that will be a different version

Reply:
Please change this to release/1.4.1 to get the proper version of fsx-lustre